<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQuizDialog</class>
 <widget class="QDialog" name="TestQuizDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="testLayout">
     <item>
      <layout class="QVBoxLayout" name="testContainer">
       <item>
        <widget class="QListWidget" name="templateTestList"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="endTestButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Закончить тест</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
